src/main/java/com/socrata/api/HttpLowLevel.java

```diff
@@ -78,6 +81,8 @@
 
     public static final GenericType<List<Object>> MAP_OBJECT_TYPE = new GenericType<List<Object>>() {};
 
+    private final ObjectMapper mapper;
```

Comment:
Agreed, but since this isn't a new field, I was considering this to be out of scope; however, I did add a new constructor signature ([line 188](https://github.com/socrata/soda-java/pull/22/files/f0ba4245234e8f0fffe1d313c50588b5723048c2#diff-478c93a691c2e2e11d027a90ef1ce5fcR188)) so that the application may provide the library with its singleton ObjectMapper